storage/tianmu/exporter/data_exporter.h

```diff
@@ -46,22 +46,22 @@ class DataExporter {
   virtual void PutRowEnd() = 0;
 
  protected:
-  int cur_attr;
-  std::vector<core::AttributeTypeInfo> deas;
-  std::vector<core::AttributeTypeInfo> source_deas;
-  fields_t _fields;
+  int cur_attr_;
```

Comment:
You should keep in mind that: 
readability is not just adding some underscore, but let other to know what you want to do just only via your code.

Therefore,` deas_` means what ? I think `attr_infos_` or something else is a better choice.

Reply:
Some obscure names have been adjusted